exe/entangler

```diff
@@ -18,12 +18,15 @@ def value_opt(options, opts, key, *args)
 end
 
 options = {}
+# rubocop:disable Metrics/BlockLength
 OptionParser.new do |opts|
-  opts.banner = %(Entangler v#{Entangler::VERSION}
+  opts.banner = <<~BANNER
```

Comment:
I changed this to use a heredoc to make it a bit easier to read, the final formatting is unchanged:

<details>
<summary>Before</summary>

```
Entangler v1.3.2

Usage:
  entangler master <base_dir> <remote_user>@<remote_host>:<remote_base_dir> [options]
  entangler master <base_dir> <other_synced_base_dir> [options]

Options:
    -i, --ignore '.git'              Ignore path when syncing, string is regex if surrounded by '/'
                                     All paths should be relative to the base sync directory.
    -p, --port PORT                  Overwrite the SSH port (usually 22)
                                     (doesn't do anything in slave mode)
        --force-polling              Forces the use of the listen polling adapter (works cross-platform, generally slower)
        --no-rvm                     Skip attempting to load RVM on remote
    -v, --verbose                    Log Debug lines
    -q, --quiet                      Don't log to stdout in master process
        --version                    Show version number
    -h, --help                       Show this message
```
<details>

<details>
<summary>After</summary>
```
Entangler v1.3.3

Usage:
  entangler master <base_dir> <remote_user>@<remote_host>:<remote_base_dir> [options]
  entangler master <base_dir> <other_synced_base_dir> [options]

Options:
    -i, --ignore '.git'              Ignore path when syncing, string is regex if surrounded by '/'
                                     All paths should be relative to the base sync directory.
    -p, --port PORT                  Overwrite the SSH port (usually 22)
                                     (doesn't do anything in slave mode)
        --force-polling              Forces the use of the listen polling adapter (works cross-platform, generally slower)
        --no-rvm                     Skip attempting to load RVM on remote
    -v, --verbose                    Log Debug lines
    -q, --quiet                      Don't log to stdout in master process
        --version                    Show version number
    -h, --help                       Show this message
```
<details>